../inc/csvcontrol.hxx:447: Fehler: invalid pure specifier (only `= 0' is allowed) before »;« token

Index: inc/bf_svx/svdobj.hxx
===================================================================
RCS file: /cvs/framework/binfilter/inc/bf_svx/svdobj.hxx,v
retrieving revision 1.5
diff -u -u -r1.5 svdobj.hxx
--- binfilter/inc/bf_svx/svdobj.hxx	9 Sep 2005 14:38:27 -0000	1.5
+++ binfilter/inc/bf_svx/svdobj.hxx	23 Oct 2006 14:01:36 -0000
@@ -265,7 +265,7 @@
 	{}
 	virtual ~SdrObjUserData();
 
-	virtual SdrObjUserData* Clone(SdrObject* pObj1) const = NULL;
+	virtual SdrObjUserData* Clone(SdrObject* pObj1) const = 0;
 	UINT32  GetInventor() const { return nInventor; }
 	UINT16  GetId() const { return nIdentifier; }
 	virtual void WriteData(SvStream& rOut);
Index: inc/svdobj.hxx
===================================================================
RCS file: /cvs/graphics/svx/inc/svdobj.hxx,v
retrieving revision 1.41
diff -u -u -r1.41 svdobj.hxx
--- svx/inc/svdobj.hxx	19 Jun 2006 14:44:11 -0000	1.41
+++ svx/inc/svdobj.hxx	23 Oct 2006 15:53:31 -0000
@@ -323,7 +323,7 @@
 	{}
 	virtual ~SdrObjUserData();
 
-	virtual SdrObjUserData* Clone(SdrObject* pObj1) const = NULL;
+	virtual SdrObjUserData* Clone(SdrObject* pObj1) const = 0;
 	UINT32  GetInventor() const { return nInventor; }
 	UINT16  GetId() const { return nIdentifier; }
 	//BFS01virtual void WriteData(SvStream& rOut);
Index: source/ui/inc/csvcontrol.hxx
===================================================================
RCS file: /cvs/sc/sc/source/ui/inc/csvcontrol.hxx,v
retrieving revision 1.13
diff -u -u -r1.13 csvcontrol.hxx
--- sc/source/ui/inc/csvcontrol.hxx	8 Sep 2005 21:17:47 -0000	1.13
+++ sc/source/ui/inc/csvcontrol.hxx	23 Oct 2006 20:57:40 -0000
@@ -444,7 +444,7 @@
 
 protected:
     /** Derived classes create a new accessible object here. */
-    virtual ScAccessibleCsvControl* ImplCreateAccessible() = NULL;
+    virtual ScAccessibleCsvControl* ImplCreateAccessible() = 0;
 };
 
 
